src/screen.cpp

```diff
@@ -1155,7 +1155,7 @@ void s_reset(screen_t *s, screen_reset_mode_t mode) {
         if (screen_width > non_space_width) {
             bool justgrey = true;
             if (cur_term && enter_dim_mode) {
-                std::string dim = tparm((char *)enter_dim_mode);
+                std::string dim = tparm(enter_dim_mode);
```

Comment:
NAK. We've been over this. This *needs* to stay because of differences in curses implementations.

Reply:
(If anyone has a better way of expressing this I'm all ears. The issue is macOS having ncurses configured as taking non-const - an option that was since removed - and NetBSD curses insisting on const, or something along those lines)